catalog/spatial/geocoding/spatial-geocoding-websearch/src/main/java/org/codice/ddf/spatial/geocoder/geonames/GeoNamesWebService.java

```diff
@@ -110,6 +110,12 @@
     return Collections.emptyList();
   }
 
+  @Override
+  public List<String> getSuggestedNames(String queryString, int maxResults)
+      throws GeoEntryQueryException {
+    return Collections.emptyList();
```

Comment:
Why do we return null and have a `no-op` comment on `GeoNamesQueryLuceneDirectoryIndex.java` but return an empty list in this implementation?

Reply:
Changed the other for consistency.

Reply:
I think we should log something if we are returning an empty list.  An empty list without an error suggests that no results were found, but in this case the method is not implemented.

Reply:
OK. 👍